django_celery_beat/models.py

```diff
@@ -545,9 +553,24 @@ def save(self, *args, **kwargs):
         self.headers = self.headers or None
         if not self.enabled:
             self.last_run_at = None
+        self._clean_expires()
         self.validate_unique()
         super(PeriodicTask, self).save(*args, **kwargs)
 
+    def _clean_expires(self):
+        if self.expire_seconds and self.expires:
+            raise ValidationError(
+                _('Only one can be set, in expires and expire_seconds')
+            )
+        if self.expires and isinstance(self.expires, (int, float)):
```

Comment:
In `install_default_entries`  function, we got float type `expires`.

Yeah, this is indeed confusing.  I wanted to finish it without making too much changed.

Because the `expires` argument of  `apply_async` is supporting DateTime or Float type, so I just did not want to the user to remember more keywords.

I  just add a comment here. (I am lazy here :)

For the readability,  these ways are may  better:
1.   Use other keywords instead of `expires` in options,  such as `expire_seconds`.
2.   Still use `expires` as same as `apply_async`,  modify `update_from_dict` to save the value into `expire_seconds` field.

What do you think? 
Do the first way is better in which we use expire_seconds in beat conf and have to add some description in the doc? @liquidpele

Reply:
I have changed into use `expire_seconds` in beat conf of options.